browser/components/markdown.styl

```diff
@@ -213,29 +213,25 @@ pre
   margin 0 0 1em
   display flex
   line-height 1.4em
-  &.flowchart, &.sequence, &.chart
-    display flex
-    justify-content center
-    background-color white
-  &.CodeMirror
-    height initial
-    flex-wrap wrap
-    &>code
-      flex 1
-      overflow-x auto
   code
     background-color inherit
     margin 0
     padding 0
     border none
     border-radius 0
+  &.CodeMirror
+    height initial
+    flex-wrap wrap
+    &>code
+      flex 1
+      overflow-x auto
   &>span.filename
-    width 100%
-    border-radius: 5px 0px 0px 0px
     margin -8px 100% 8px -8px
```

Comment:
Suggested change to fix a small display bug:
`z-index: 10`
Before:
![image](https://user-images.githubusercontent.com/12984316/45588105-8165b180-b939-11e8-9231-69c7aca1ba15.png)
After:
![image](https://user-images.githubusercontent.com/12984316/45588109-99d5cc00-b939-11e8-996c-01a00651038e.png)

Reply:
@ZeroX-DG I removed the errors for every diagram (chart, flowchart, mermaid and sequence).
I've also fixed a bug with the line number...

But I can't replicated the visual bug you are getting...
Here what I get:
![screenshot](https://user-images.githubusercontent.com/587742/45588325-5d8f7500-b913-11e8-960e-f9b8e579426d.jpg)

Here my preferences:
![screenshot](https://user-images.githubusercontent.com/587742/45588449-39cd2e80-b915-11e8-843a-4d1244e56191.jpg)

Reply:
You preview font size is too small, you can set it to `24` and you will see the problem

Reply:
Here my fix:
![screenshot](https://user-images.githubusercontent.com/587742/45588698-eeb51a80-b918-11e8-9093-e7a57d78cf45.png)